plone/app/upgrade/v51/configure.zcml

```diff
@@ -133,11 +133,23 @@ Add image scaling options to image handling control panel.
         destination="5107"
         profile="Products.CMFPlone:plone">
 
+        <gs:upgradeDepends
+            title="Run to51beta4 upgrade profile."
+            description="
+New metadata column `mime_type`.
```

Comment:
Just cosmetics, but is there any good reason to add the description in a new line?